src/control/cmd/dmg/system.go

```diff
@@ -207,14 +207,14 @@ func displaySystemAction(log logging.Logger, results system.MemberResults) error
 type systemStopCmd struct {
 	logCmd
 	connectedCmd
-	Force bool `long:"force" description:"Force stop DAOS system members"`
 }
 
 // Execute is run when systemStopCmd activates
 //
 // Perform prep and kill stages with stop command.
 func (cmd *systemStopCmd) Execute(_ []string) error {
-	req := client.SystemStopReq{Prep: true, Kill: true, Force: cmd.Force}
+	// always force stop while daos_io_server shutdown is broken.
```

Comment:
Would prefer to have this comment be a TODO/FIXME with an associated 1.2 ticket so that we don't forget about it.

Reply:
this PR isn't going to master, so , there is no associated fix necessary

Reply:
Ah, right. OK, so the ticket isn't necessary but we should still leave the dummy `--force` option in place for consistency.

Reply:
Does this problem only happen on release/0.9 then?

Reply:
No, these are two different things. This PR concerns the `dmg system stop` administrative command which is used for stopping the data plane from the control plane. The change made was to always force-stop (i.e. SIGKILL) the ioservers, because the ioserver signal handling will not be fixed for 1.0. This command does not stop the control plane.

PR #2355 has to do with what the control plane process does when it receives a signal (e.g. SIGINT, SIGTERM). There was an earlier attempt to forward "graceful" signals on to the ioservers, but as noted in the previous paragraph that signal handling is broken in the ioserver. That work was never landed to release/0.9, so reverting it only needed to happen on master. With this PR, the ioservers are always sent SIGKILL regardless of the signal received by the control plane process.